tests/android/browserWithCustomPort.spec.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2020 Microsoft Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { androidTest as test, expect } from './androidTestWithCustomPort';
```

Comment:
instead of mirroring the other tests, lets either do parameterised tests with the port and default port or just add a single test which connects to some other port. There you either have to launch the adb daemon manually on another port or inside the test you add a small tcp proxy which seems the easiest.

Reply:
Have done a parameterised test check with the port.